Tasks/DecryptFile/decrypt.ps1

```diff
@@ -0,0 +1,60 @@
+#
+#  Copyright (c) Microsoft. All rights reserved.  
+#  Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+param(
+    [string]$cipher,
+    [string]$inFile,
+    [string]$passphrase,
+    [string]$outFile,
+    [string]$cwd
+)
+
+Write-Verbose "Entering script DecryptFile.ps1"
+Write-Host "cipher = $cipher"
```

Comment:
If this is going in update 1 and visible for on prem customers, then user facing strings (write-host, write-warning, write-error) should be localized. You can use the Get-LocalizedString cmdlet and I can generate and add the strings to the resx file.